lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -320,13 +320,27 @@ export class UserAgentApplication {
     }
 
     // #endregion
+
     /**
      * Use when initiating the login process by redirecting the user's browser to the authorization endpoint.
      * @param {@link (AuthenticationParameters:type)}
      */
     loginRedirect(userRequest?: AuthenticationParameters): void {
+        // Check if userRequest is empty
+        if (!userRequest) {
+            userRequest = {};
+        }
+        
+        // Check if request scopes are null
+        if (!userRequest.scopes) {
+            userRequest.scopes = [];
+        }
+
+        // Append clientId to scopes by default for login calls
+        userRequest.scopes = ScopeSet.appendScopes(userRequest.scopes, [this.clientId]);
```

Comment:
Can we change this to openid and profile instead of clientId since we will replace clientId later anyway?

Reply:
Done!